src/apiCalls.js

```diff
@@ -14,9 +14,36 @@ let pageData = {
 };
 
 // API CALLS
-const fetchUsers = () => fetch('http://localhost:3001/api/v1/users')
-const fetchRecipes = () => fetch('http://localhost:3001/api/v1/recipes')
-const fetchIngredients = () => fetch(`http://localhost:3001/api/v1/ingredients`)
+const getUsers = () => fetch('http://localhost:3001/api/v1/users')
+const getRecipes = () => fetch('http://localhost:3001/api/v1/recipes')
+const getIngredients = () => fetch(`http://localhost:3001/api/v1/ingredients`)
+const updateRecipe = (userID, recipeID, request) => {
+  const body = {
+    userID,
+    recipeID
+  };
+
+  return fetch('http://localhost:3001/api/v1/usersRecipes', {
+    method: `${request}`,
+    body: JSON.stringify(body),
+    headers: {
+      'Content-Type': 'application/json'
+  }})
+}
+
+const getUsersAfterUpdate = (userID, recipeID, e) => {
+  return getUsers()
+          .then(response => {
+            if(response.message) {console.log("get response",response.message)}
+            return response.json();
+          })
+          .then(data => {
+            const foundUser = data.users.find(user => user.id === userID);
+            currentUser = foundUser;
+            toggleSavedButtons(e, recipeID, currentUser)
```

Comment:
invoke renderTagsAfterFetch here after you have updated currentUser to get rid of the deleting but and don't invoke it in the other two places where it is originally invoked

Reply:
done